tutorials/phasorpy_introduction.py

```diff
@@ -168,10 +168,9 @@
 # Pixels with low intensities are commonly excluded from analysis and
 # visualization of phasor coordinates:
 
-# TODO: replace this with a ``phasor_mask`` function?
-mask = mean > 1
-real = real[mask]
-imag = imag[mask]
+from phasorpy.phasor import phasor_threshold
+
+mean, real, imag = phasor_threshold(mean, real, imag, 1)
```

Comment:
Probably doesn't matter, but I think the equivalent to filter `mean>1` is `phasor_threshold(mean, real, imag, 2)`, no?